definitions/procedures/iptables/add_chain.rb

```diff
@@ -2,15 +2,15 @@ module Procedures::Iptables
   class AddChain < ForemanMaintain::Procedure
     metadata do
       label :iptables_add_chain
-      for_feature :maintenance_mode
+      for_feature :iptables
       description 'Add chain to iptables'
       tags :pre_migrations
       advanced_run false
       after :sync_plans_disable
     end
 
     def run
-      feature(:maintenance_mode).perform_action(:iptables, 'add_chain')
+      feature(:iptables).perform_action('add_chain')
```

Comment:
Why the action is called indirectly? Are there any benefits over `feature(:iptables).add_chain`?